_docs/kustomize/akash-provider/deployment.yaml

```diff
@@ -223,6 +223,54 @@ spec:
                   key: mem-price-max
                   optional: true
```

Comment:
Is the old stuff still relevant?  `mem-price-min`, etc...

Reply:
The original pricing algorithm is still present in the code. I'm assuming its used in some of the e2e tests.